### PR TITLE
add some suggestions for good plan practices

### PR DESCRIPTION
**Pull request by smith**
gif-keyboard-5502027721044281789

### CODE REVIEW

File: www/source/docs/reference/plan-syntax.html.md [hunk 1]
```diff
@@ -611,3 +611,15 @@ $ hab start core/redis --config-from /src
 This would take the configuration and hooks from /src, rather than from the
 package you have previously built. When the configuration is as you want it,
 do a final rebuild of the package.
+
+## Good Practices for Plan Development
+
+* If you plan has "internal" dependencies, for example, if you're building a
```
Comment (mwrock) [inline www/source/docs/reference/plan-syntax.html.md]: your plan :)

Comment (smith) [inline www/source/docs/reference/plan-syntax.html.md]: fixed

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/200124544).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/200124544).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**